3_javascript/javascript1/week3/script.js

```diff
@@ -0,0 +1,81 @@
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+const nameToRemove = "Ahmad";
+
+const indexOfnameToRemove = names.indexOf(nameToRemove);
```

Comment:
You need to implement the code that removes 'nameToRemove' from the 'names' array. Consider using different array methods.